code/game/gamemodes/endgame/supermatter_cascade/universe.dm

```diff
@@ -11,9 +11,13 @@
 	return 0
 
 /datum/universal_state/supermatter_cascade/OnTurfChange(var/turf/T)
-	var/turf/space/spess = T
-	if(istype(spess))
-		spess.overlays += "end01"
+	if(T.name == "space")
+		T.overlays += "end01"
+		T.underlays -= "end01"
+	else
+		T.overlays -= "end01"
```

Comment:
But if this is a change from a space turf to another kind of turf, surely it doesn't already have the underlay?